src/apps/weblib/js-api/vizzu.js

```diff
@@ -71,10 +71,14 @@ export default class Vizzu {
       try {
         return f(...params);
       } catch (e) {
-        let address = parseInt(e, 10);
-        let cMessage = this.module._vizzu_errorMessage(address);
-        let message = this.module.UTF8ToString(cMessage);
-        throw new Error("error: " + message);
+        if (Number.isInteger(e)) {
```

Comment:
Please put a line about this in the Changelog's ureleased/fixed section

Reply:
Is it good? https://github.com/vizzuhq/vizzu-lib/pull/224/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR19